test/python/circuit/test_parameters.py

```diff
@@ -384,6 +385,24 @@ def test_transpiling_multiple_parameterized_circuits(self):
 
         self.assertTrue(len(job.result().results), 2)
 
+    def test_transpile_across_optimization_levels(self):
+        """Verify parameterized circuits can be transpiled with all default pass managers."""
+
+        qc = QuantumCircuit(5, 5)
+
+        theta = Parameter('theta')
+        phi = Parameter('phi')
+
+        qc.rx(theta, 0)
+        qc.x(0)
+        for i in range(5-1):
+            qc.rxx(phi, i, i+1)
+
+        qc.measure(range(5-1), range(5-1))
+
+        for i in [0, 1, 2, 3]:
+            transpile(qc, FakeOurense(), optimization_level=i)
```

Comment:
I get the tests are to verify we don't raise, but I think it's good practice to at least assert something in a test. Even if it's just that the type output from transpile is a circuit